client/components/banner-notice/index.tsx

```diff
@@ -80,7 +80,7 @@ function BannerNotice( props: BannerNoticeProps ): JSX.Element {
 
 	return (
 		<Notice { ...noticeProps }>
-			<Flex align="flex-start" justify="flex-start">
```

Comment:
This change the change below was originally submitted to https://github.com/Automattic/woocommerce-payments/pull/5839 but given that PR will be on-hold for some time, I cherry picked the commit over to this so the notice looks correct.